### ISSUE TITLE
Please remove ligatures from source code browser

### ISSUE THREAD

**Issue by 2mol** [haskell/hackage-server#841]
Hi! First of all, despite some kinks I really love hackage. When I discovered that little `# Source` button my mind was a little bit blown.

There is one painful thing about it though, and it's that hackage generates these cute 1-character ligatures for some symbols. Examples include `∷` instead of `::`, `⊕` for `++`, `μ` for whatever, etc.

I always found these to be potentially extremely confusing for newcomers:

cons:

- I can't type these
- I might not know what they mean
- They are not even valid code

Point 3 is the most important one: When I want to copy-paste code (super useful), I get the WEIRDEST error message. I just spent a stupid amount of time realizing a problem with `∷`. The error message?

```
Parse error: module header, import declaration
or top-level declaration expected.
```

pros:

- They look pretty and are a cute gimmick.

In conclusion: they really provide zero value in my opinion.

I volunteer for creating a PR, but let's have a discussion here first, I'm curious to hear if there are any counter-arguments.

**Comment by phadej** [comment]
Can you give an example? I think those are actually a source of that particular package using https://downloads.haskell.org/~ghc/8.8.1/docs/html/users_guide/glasgow_exts.html#extension-UnicodeSyntax

**Comment by 2mol** [comment]
You're right! Damn, sorry for complaining here then, will close.

Here was the example I was using: http://hackage.haskell.org/package/wai-cors-0.2.7/docs/src/Network.Wai.Middleware.Cors.html

I see the UnicodeSyntax pragma now. Really not a fan, but not hackage's problem. Thanks for the quick answer!
